config/routes.rb

```diff
@@ -70,7 +70,7 @@ def matches?(request)
       end
     end
 
-    if Rails.env.test?
+    if Rails.env.test? || Rails.env.development?
```

Comment:
I don't think this should be here.

Reply:
Ooops, that was me trying to bypass login when pre-prod was down 😆